client/me/purchases/manage-purchase/payment-method-selector/tos-text.tsx

```diff
@@ -12,7 +13,11 @@ export default function TosText() {
 					components: {
 						tosLink: (
 							<a
-								href={ localizeUrl( 'https://wordpress.com/tos/' ) }
+								href={
+									isAkismetCheckout()
```

Comment:
This component is used in the context of updating a payment method on an individual purchase. The check for `isAkismetCheckout` will always be false here since it checks for a checkout path. I think we'll need to check what the current product is instead?

Nice catch on updating this component as well.

Reply:
Thanks for catching that, got that fixed up!